modules/auxiliary/dos/osx/darwin_nuke_icmp.rb

```diff
@@ -0,0 +1,60 @@
+##
```

Comment:
*One of these things is not like the others
One of these things just doesn't belong
Can you tell which thing is not like the others
By the time I finish my song?*